atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -298,9 +299,13 @@ public static String internalTableName(TableReference tableRef) {
         return tableName.replaceFirst("\\.", "__");
     }
 
+    /** @deprecated uses TableReference.createUnsafe, which is itself deprecated.
+     *
+     */
+    @Deprecated
```

Comment:
Not sure we can deprecate this method.  As in we will always need to have some mapping from persisted tables to TableReferences.  That said I would buy an argument from removing this from the abstract KVS impl and have one per KVS.